orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -188,6 +188,15 @@ class ORTTrainerOptions(object):
                             'type': 'integer',
                             'min': 0,
                             'default': 0
+                        },
+                        'propagate_cast_ops_level': {
```

Comment:
Update schema here https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/orttrainer_options.py#L26

Here https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/orttrainer_options.py#L294

And here https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/orttrainer_options.py#L500

Otherwise, unit tests wont match and ORTTrainer users will be confused.

Current status is that although we wont GA ORTTrainer, we will still have to support users, so keeping the code coherent is important for the long term

Reply:
I  have already added graph_transformer.propagate_cast_ops_allow option in this file in two places (L26 and L500) . What needs to be changed in L294?